### PR TITLE
Resolve distorted UI issues

### LINKED ISSUE #5015: Distorted UI on multi signature views
### Actual behavior
- The checkmark icon is broken.
image

- Distorted UI on multi signature summary page. This was observed during a staking transaction from a multi signature account.
image

- On staking a validator from a multi sig account. On signing the transaction and coping the transaction object, the staking queue was not cleared and because of this, switching to the other account in order to complete the transaction had the pending stakes confirmation modal shown up. Which resulted to a poor UX

### Steps to reproduce
- Click on any copy text button (the checkmark icon would be broken)
- Initiate the signing process of a staking transaction from a multi-signature account
- Try to sign the account as a member of the multi-sig group (its observe the summary page is broken and you get the confirm modal before when you try switching account to another)

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #5015, #5047, #5015, #5042 and #5034

### How was it solved?

- Use correct icon for checkmark
- Updated grammar for suggested text
- Updated css to handle the multisignature summary view

### How was it tested?

- Run the app and review the grammar changes
- Create multi-signature account and review the checkmark icon
- Sign a multi-signature transaction for all types supported and review the summary page to be not distorted

### CODE REVIEW

**Review by ManuGowda — COMMENTED** [review]
Screenshot 2023-06-05 at 4 27 07 PM

for generic case, the pretty JSON should be rendered properly

File: src/modules/blockchainApplication/manage/components/RemoveApplicationSuccess/RemoveApplicationSuccess.js [hunk 1]
```diff
@@ -27,10 +27,7 @@ const RemoveApplicationSuccess = ({ history, sharedData: { application } }) => {
           {t('You can always add')} <a onClick={handleAddApplication}>{application.chainName}</a>{' '}
           {t('again to your application list.')}
         </p>
-        <PrimaryButton
-          className={`${styles.button}`}
-          onClick={onRemoveSuccess}
-        >
+        <PrimaryButton className={`${styles.button}`} onClick={onRemoveSuccess}>
```
Comment (eniolam1000752) [inline src/modules/blockchainApplication/manage/components/RemoveApplicationSuccess/RemoveApplicationSuccess.js]:
```suggestion
        <PrimaryButton className={styles.button} onClick={onRemoveSuccess}>
```

**Review by eniolam1000752 — CHANGES_REQUESTED** [review]
## UI comments

- For a cross chain token transfer, I think it would be cool if the summary shows recipient chain and its logo
image

File: src/modules/blockchainApplication/manage/components/RemoveApplicationSuccess/RemoveApplicationSuccess.js (hunk 1 above)
Comment (ikem-legend) [reply src/modules/blockchainApplication/manage/components/RemoveApplicationSuccess/RemoveApplicationSuccess.js]: Done

**Review by eniolam1000752 — APPROVED** [review]

File: src/modules/transaction/components/TransactionDetails/messageFee.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import TokenAmount from '@token/fungible/components/tokenAmount';
+import TransactionDetailsContext from '../../context/transactionDetailsContext';
+import ValueAndLabel from './valueAndLabel';
+import styles from './styles.css';
+
+const Fee = ({ t }) => {
```
Comment (ikem-legend) [inline src/modules/transaction/components/TransactionDetails/messageFee.js:7]: This should be renamed to MessageFee

**Review by ikem-legend — APPROVED** [review]
Looks good to me. Added a minor comment

**Pull request merged**